src/components/CreateNewCard.js

```diff
@@ -0,0 +1,56 @@
+import React, { useState, useRef } from "react";
+import "./CreateNewCard.css";
+import "bootstrap/dist/css/bootstrap.min.css";
+import Button from "react-bootstrap/Button";
+import Form from "react-bootstrap/Form";
+
+const defaultCardData = {
+  message: "",
+};
+
+const CreateNewCard = ({ onCardUpdate }) => {
```

Comment:
I suggest striving to be consistent with how things are named across a project. The files that hold the forms that create new cards and boards have different naming schemes `CreateNewCard` vs `NewBoardForm`. I would pick one format and use it for both, either totally works.